### ISSUE TITLE
Make the espeak-ng sound less lame

### ISSUE THREAD

**Issue by sthibaul** [brailcom/speechd#884]
### Expected behavior

From @albertotirla :

“make it sound less lame, for example make it sound like nvda does it or something of that nature”

Not sure what nvda is doing differently, it would need investigation, as AFAIK spd mostly uses default espeak-ng parameters. I'd argue that if the espeak-ng defaults are bad, they should be changed there, but at least we can include whatever nvda tuning uses for older releases of espeak-ng.

**Comment by sthibaul** [comment]
“I don't have issues with espeak for example, but I heard it from someone who used nvda with inflection turned up to 100, and the first thing they said when I turned on orca on their vm, was that espeak sounds different and more compressed”

**Comment by sthibaul** [comment]
> wow raising resample.quality from 4 to 10 in /etc/pipewire/client.conf, /etc/pipewire/pipewire-pulse.conf makes a BIG improvement in espeak-ng sound

Ouch, I'd say it's a big pipewire issue...

I now somehow unsterstand why possibly a lot of people don't have the issue, since that probably depends on the audio card capabilities. In `pw-top`, the RATE columns matches the application column on my laptop, so I guess I have no resampling and thus no issue.

I wonder if once we use the native pipewire API, there will be a programmatic way to request good quality resampling

**Comment by sthibaul** [comment]
> I wonder if once we use the native pipewire API, there will be a programmatic way to request good quality resampling

I didn't get it to work : https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/4800 

But we could stuff some configuration bit, by installing in `/etc/pipewire/client.conf.d/speechd-quality.conf` :

```
node.rules = [
	{
		matches = [ { media.role = "Accessibility" } ]
		actions = { update-props = { resample.quality = 10 } }
	}
]
```

@alex19EP : Is 10 a good-enough value? does the maximum (14) give even better result? pipewire people tend to say that it's useless cpu eating, unless you do perceive a difference between 10 and 14?
